src/popup/pages/Lock/index.js

```diff
@@ -107,6 +121,109 @@ class LockPage extends React.Component {
+                    value={this.state.confirmText}
+                    onTextInput={this.onTextInput}
+                />
+            </div>)
+    }
+    renderActionBtn = () => {
+        return (
+            <div className={"account-info-btn-container"}>
+                <Button
+                    content={getLanguage('confirm')}
+                    onClick={this.onConfirmRestore}
+                    propsClass={"account-common-btn"}
+                    disabled={!this.state.deleteTagStatus}
+                />
+                <Button
+                    content={getLanguage('cancel')}
+                    propsClass={"account-common-btn account-common-btn-cancel"}
+                    onClick={this.onCloseModal}
+                />
+            </div>
+        )
+    }
+    renderChangeModal = () => {
+        return (<TestModal
+            ref={this.modal}>
+            <div className={'account-change-name-container'}>
+                <div className={"account-change-title-container"}>
+                    <p className={
+                        cx({ "account-change-name-title": true })
+                    }>{getLanguage('confirm_restore_tip')}</p>
+                </div>
+                {this.renderInput()}
+                {this.renderActionBtn()}
+            </div>
+        </TestModal>)
+    }
+    checkDeleteTagStatus = () => {
+        let realText = trimSpace(this.state.confirmText)
+        let deleteTag = getLanguage("deleteTag")
+        let checkStatus = realText === deleteTag
+        this.callSetState({
+            deleteTagStatus: checkStatus
+        })
+    }
+    onConfirmRestore = () => {
+        if (this.state.deleteTagStatus) {
+            //1, delete extension storage
+            clearStorage()
+
+            //2, delete local storage except net-config
+            clearLocalExcept(NET_WORK_CONFIG)
+            //3, clear all redux
+            this.props.resetWallet()
+            //4, get netconfig and update net-reducer
+            let netConfig = getLocal(NET_WORK_CONFIG)
+            if (netConfig) {
+                netConfig = JSON.parse(netConfig)
+                this.props.updateNetConfigList(netConfig)
+            }
+
+            this.props.history.push({
+                pathname: "/welcome",
+            });
+        }
+    }
+    showConfirmModal = () => {
+        this.modal.current.setModalVisable(true)
+    }
+    onCloseModal = () => {
+        this.modal.current.setModalVisable(false)
+    }
+    onClickRestore = () => {
+        let title = getLanguage('prompt')
+        let content = ""
+        let cancelText = getLanguage('confirmRestore')
+        let confirmText = getLanguage('cancelRestore')
+        let tipImgSrc = txFailed
+        content = [
+            getLanguage('restore_tip_1'),
+            getLanguage('restore_tip_2'),]
+        ConfirmModal.show({
+            title,
+            content,
+            cancelText,
+            confirmText,
```

Comment:
Rename.
- cancelText and confirmText are reversed.
- confirmRestore should be confirmReset.

```suggestion
        let confirmText = getLanguage('confirmReset')
        let cancelText = getLanguage('cancelReset')
        let tipImgSrc = txFailed
        content = [
            getLanguage('restore_tip_1'),
            getLanguage('restore_tip_2'),]
        ConfirmModal.show({
            title,
            content,
            confirmText,
            cancelText,
```

Reply:
The button reversed is for users to focus on it. The left side is for confirmation. Generally, users are more accustomed to confirming on the right side, but this operation is very sensitive, so I hope to put it on the left to let users confirm. The reason for the name here is because most of the encapsulated components are canceled on the left, so the name is like this。 but i will change restore to reset , it's better